<?xml version="1.0"?>

<launch>
  <node name="projection" pkg="cob_people_object_detection_tensorflow" type="projection.py" output="screen">
    <rosparam command="load" file="$(find cob_people_object_detection_tensorflow)/launch/projection_params.yaml"/>
  </node>
</launch>
